### DISCUSSION TITLE
Change .ini settings

### DISCUSSION THREAD

**Discussion by NeoDragonXDA** [wolveix/satisfactory-server#206] [Q&A] [answered]
Is there any way to change engine.ini settings ? They get overwritten when recreating container so there's no way to load them at start.

**Comment by wolveix** [comment, marked as answer]
Hey!

Per this issue: https://github.com/wolveix/satisfactory-server/issues/177, you can mount your own versions of these files to `/home/steam/file.ini` and the container will use those versions instead. We do already set all of the well-known recommended changes for servers, and most values that you'd want to change are configurable via environment variables. If there's something you wanted to change outside of what we natively support, I'd love to know more so that we could potentially implement it for others :)
